src/nnue/network.h

```diff
@@ -34,6 +34,7 @@
 
 namespace Stockfish::Eval::NNUE {
 
+struct AccumulatorRefreshEntry;
```

Comment:
one more note

I'd consider a different name, as this is only coupled with refreshing the accumulator by the current usage. The scope of this struct is broader. `ComputedAccumulator`, `AccumulatorCacheEntry`.

It'd also be nice to have a comment pointing to the origin of the idea.